internal/keepers/util_test.go

```diff
@@ -180,6 +180,20 @@ func Test_observationsToUpkeepKeysError(t *testing.T) {
 				ktypes.UpkeepIdentifier("03"), // Error in both
 			},
 		},
+		{
+			BlockKey: chain.BlockKey("-1"), // Should be non negative
+			UpkeepIdentifiers: []ktypes.UpkeepIdentifier{
+				ktypes.UpkeepIdentifier("2"),
+				ktypes.UpkeepIdentifier("3"),
+			},
+		},
+		{
+			BlockKey: chain.BlockKey("1"), // Should be non negative
+			UpkeepIdentifiers: []ktypes.UpkeepIdentifier{
+				ktypes.UpkeepIdentifier("-2"),
```

Comment:
Should this line should be commented to state that the UpkeepIdentifier should be non-negative?